pages/linux/pacman.md

```diff
@@ -6,32 +6,32 @@
 
 - Synchronize and update all packages:
 
-`sudo pacman --sync --refresh --sysupgrade`
+`sudo pacman -Syu`
```

Comment:
All short options should be described via square brackets where possible here and in all translations.